### PR TITLE
MINOR: Log more information when producer snapshot is written

### PR DESCRIPTION
**Pull request by dajac**
This patch logs more information when a producer snapshot is written to the disk.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/log/Log.scala [hunk 1]
```diff
@@ -2011,7 +2011,11 @@ object Log extends Logging {
       logDirFailureChannel,
       config.messageFormatVersion.recordVersion,
       s"[Log partition=$topicPartition, dir=${dir.getParent}] ")
-    val producerStateManager = new ProducerStateManager(topicPartition, dir, maxProducerIdExpirationMs)
+    val producerStateManager = new ProducerStateManager(
+      topicPartition,
+      dir,
+      maxProducerIdExpirationMs,
+      time)
```
Comment (ijuma) [inline core/src/main/scala/kafka/log/Log.scala]: Nit: is this really easier to read than just adding the `time` parameter in the same line?

**Review by ijuma — APPROVED** [review]
LGTM, thanks!

File: core/src/main/scala/kafka/log/Log.scala (hunk 1 above)
Comment (dajac) [reply core/src/main/scala/kafka/log/Log.scala]: Both are equivalent for me. I found that the line was getting too long with the addition of `time` so I broke it down that way to stay inline with `LogLoader.load` below. Would you prefer to keep it on one line instead?

Comment (ijuma) [reply core/src/main/scala/kafka/log/Log.scala]: I prefer one line in this case, but I'm fine either way. :)

**Review by lbradstreet — APPROVED** [review]
LGTM, thanks!

File: core/src/main/scala/kafka/log/Log.scala (hunk 1 above)
Comment (dajac) [reply core/src/main/scala/kafka/log/Log.scala]: Reverted to using one line.

**Comment by dajac** [comment]
@ijuma @lbradstreet Thanks for your review. Merging to trunk.

**Pull request merged**